estimation_tracker/tracker/templatetags/custom_tags.py

```diff
@@ -0,0 +1,43 @@
+from django import template
+import datetime
+import math
+
+register = template.Library()
+
+
+def convert_time_to_seconds(time: datetime.time) -> int:
+    seconds = (time.hour * 60 + time.minute) * 60 + time.second
```

Comment:
There is already a method doing that for you. 
https://stackoverflow.com/questions/5522031/convert-timedelta-to-total-seconds

Reply:
I will change this then and implement that method.